frontend/src/components/pages/ManageStoryPage.tsx

```diff
@@ -245,7 +299,8 @@ const ManageStoryPage = () => {
             isDisabled={
               title === tempTitle &&
               description === tempDescription &&
-              youtubeLink === tempYoutubeLink
+              youtubeLink === tempYoutubeLink &&
+              level === tempLevel
```

Comment:
cancel button remains disabled even when level has changed, can you add this condition to the cancel button too? (about line 290)